lib/c.zig

```diff
@@ -583,6 +583,33 @@ fn clone() callconv(.Naked) void {
                 \\ restore
             );
         },
+        .loongarch64 => {
+            // __clone(func, stack, flags, arg, ptid, tls, ctid)
+            //           a0,    a1,    a2,  a3,   a4,  a5,   a6
+            // sys_clone(flags, stack, ptid, ctid, tls)
+            //              a0,    a1,   a2,   a3,  a4
+            asm volatile (
+                \\ # Save function pointer and argument pointer on new thread stack
```

Comment:
This is missing stack alignment: https://git.musl-libc.org/cgit/musl/tree/src/thread/loongarch64/clone.s#n10

Reply:
Fixed.